src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.cs

```diff
@@ -1676,18 +1676,6 @@ internal protected virtual CSharpSyntaxNode GetBindableSyntaxNode(CSharpSyntaxNo
                     case SyntaxKind.AnonymousObjectMemberDeclarator:
                         return GetBindableSyntaxNode(parent);
 
-                    case SyntaxKind.DeclarationExpression:
-                    case SyntaxKind.TupleExpression:
```

Comment:
Confirmed we need to remove this code. Thanks

---
In reply to: [95018184](https://github.com/dotnet/roslyn/pull/16153#discussion_r95018184) [](ancestors = 95018184)